packages/sit-onyx/src/components/OnyxMobileNavButton/OnyxMobileNavButton.vue

```diff
@@ -14,37 +15,49 @@ const emit = defineEmits<{
   "update:open": [isOpen: boolean];
 }>();
 
-defineSlots<{
+const slots = defineSlots<{
   /**
-   * Slot for the menu content when its open.
+   * Slot for the menu content when it's open.
    */
   default(): unknown;
+  /**
+   * Slot for a headline of the menu content when it's open
+   */
+  headline?(): unknown;
 }>();
 </script>
 
 <template>
-  <div>
+  <div class="onyx-mobile-nav-button">
     <button
       type="button"
-      class="onyx-mobile-nav-button"
-      :class="{ 'onyx-mobile-nav-button--active': props.open }"
+      class="onyx-mobile-nav-button__trigger"
+      :class="{ 'onyx-mobile-nav-button__trigger--active': props.open }"
       :aria-label="props.label"
       @click="emit('update:open', !props.open)"
     >
       <OnyxIcon :icon="props.open ? x : props.icon" />
     </button>
 
-    <div v-if="props.open" class="onyx-mobile-nav-button__menu">
-      <div class="onyx-mobile-nav-button__content">
-        <slot></slot>
-      </div>
+    <template v-if="props.open">
+      <div class="onyx-mobile-nav-button__flyout">
+        <div
+          class="onyx-mobile-nav-button__menu"
+          :class="{ 'onyx-mobile-nav-button__menu--no-headline': !slots.headline }"
+        >
+          <OnyxHeadline is="h2" v-if="slots.headline" class="onyx-mobile-nav-button__headline">
```

Comment:
Why do we nee a slot for this? Can't we just pass the OnyxHeadline via the slot?

Reply:
there is the reason of having the headline sticky on top with the padding so the content is not visible behind if when you scroll, and if there is no headline the component needs to set additional padding-top

if you want to talk those things through, we can phone :) I had some worries about that approach aswell..

should I ask @jannick-ux  if we can drop the sticky headline?

Reply:
The sticky headline is fine for me, its just a little weird for me to have a headline slot to pass the text when the default slot is simply below it so we could also remove the headline slot and place the OnyxHeadline in the default slot inside the OnyxNavBar.

Or maybe use a property for the headline :)

Reply:
I'll do a prop then :)

Reply:
done